hashtagsv2/hashtags/views.py

```diff
@@ -161,4 +163,72 @@ def top_user_statistics_data(request):
         'usernames': usernames,
         'edits_per_user': edits_per_user
     }
+    return JsonResponse(data)
+
+def time_statistics_data(request):
+    request_dict = request.GET.dict()
+
+    edits_array = []
+    time_array = []
+
+    hashtags = hashtag_queryset(request_dict)
+    earliest_date = hashtags.earliest('timestamp').timestamp.date()
```

Comment:
We could save ourselves from making this query and the one below it like this: https://github.com/WikipediaLibrary/hashtags/blob/master/hashtagsv2/hashtags/views.py#L46. Once we've made a query and got our list of objects, just select the first and last ones, then we only make one db query instead of 3.

Reply:
Each database query takes time to execute, so if we can reduce the number we need to make we should 😁

Reply:
Yeah! @Samwalton9 that makes sense. Thanks for pointing it out. :)